File: SCDproiect-main/src/main/java/edu/utcn/timetracking/server/timeTracking/TimeTracking.java
package edu.utcn.timetracking.server.timeTracking;

import edu.utcn.timetracking.server.employee.Employee;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import org.hibernate.annotations.CreationTimestamp;

import javax.persistence.*;
import java.util.Date;

@Entity
@Data
public class TimeTracking {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @ApiModelProperty(hidden = true)
    private Integer idTime;
    private Float minutes;
    private Date date;

    @ManyToOne()
    @JoinColumn(name = "employee_id", nullable = false, foreignKey = @ForeignKey(name="employee_id"))
    private Employee employee;
}

File: WinFormsApp1/WinFormsApp1/TimeTracking.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace WinFormsApp1
{
    class TimeTracking
    {
        public int idTime { get; set; }

        public double minutes { get; set; }

        public DateTime date { get; set; }

        //public int idEmployee { get; set; }

        public Employee employee { get; set; }
        public int getId(Employee employee) { return employee.id;  }
       

    }
}
